pretyping/evarsolve.mli

```diff
@@ -179,4 +179,5 @@ val remove_instance_local_defs :
   evar_map -> Evar.t -> 'a list -> 'a list
 
 val get_type_of_refresh :
-  ?polyprop:bool -> ?lax:bool -> env -> evar_map -> constr -> evar_map * types
+  ?polyprop:bool -> ?lax:bool -> env -> evar_map * 'a -> constr
+  -> (evar_map * 'a) * types
```

Comment:
This function shouldn't have the `'a` part

Reply:
But this is how `get_type_of_refresh` is being used in rewrite.ml.

Reply:
Should I change how it is being called?

Reply:
yes